doctor_visits/delphi_doctor_visits/download_claims_ftp_files.py

```diff
@@ -51,9 +51,13 @@ def change_date_format(name):
     name = '_'.join(split_name)
     return name
 
-def download(ftp_credentials, out_path, logger):
+def download(ftp_credentials, out_path, logger, issue=None):
     """Pull the latest raw files."""
-    current_time = datetime.datetime.now()
+    if issue is None:
+        current_time = datetime.datetime.now()
+    else:
+        current_time = datetime.datetime.strptime(issue, "%Y-%m-%d").replace(hour=23, minute=59, second=59)
+
     logger.info("starting download", time=current_time)
```

Comment:
Is there something to tell the difference between a patch and a regular run? Just wondering if we should add log message to indicate the difference